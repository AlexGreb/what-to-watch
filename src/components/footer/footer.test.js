import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './footer.jsx';

describe(`Footer`, () => {
  it(`Should render correctly footer`, () => {
    const tree = renderer.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

});

