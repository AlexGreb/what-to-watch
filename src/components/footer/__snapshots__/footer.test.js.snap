// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render correctly footer 1`] = `
<DocumentFragment>
  <footer
    class="page-footer"
  >
    <div
      class="logo"
    >
      <a
        class="logo__link logo__link--light"
      >
        <span
          class="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          class="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          class="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      class="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
